### PR TITLE
[16.0][MIG] base_global_discount: 16.0

### PR DESCRIPTION
**Pull request by ferran-S73**

### CODE REVIEW

File: base_global_discount/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        relation="supplier_global_discount_rel",
+        column1="partner_id",
+        column2="global_discount_id",
+        string="Purchase Global Discounts",
+        domain=[("discount_scope", "=", "purchase")],
+    )
+    # HACK: Looks like UI doesn't behave well with Many2many fields and
+    # negative groups when the same field is shown. In this case, we want to
+    # show the readonly version to any not in the global discount group.
+    # TODO: Check in future versions if it's fixed
```
Comment (rousseldenis) [inline base_global_discount/models/res_partner.py]: @ferran-S73 Can you check this ?

Comment (ferran-S73) [reply base_global_discount/models/res_partner.py]: @rousseldenis I did a couple of tests and it appears to be working fine with just one field

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**